hack/charts/namespaced/crd_chart/templates/crds.yaml

```diff
@@ -1,6 +1,8 @@
 apiVersion: apiextensions.k8s.io/v1beta1
```

Comment:
There is no way we should be updating this manually? This seems absurd, because it would become out of date for every new feature?

Reply:
updated the CRDs, the operator would still have to be manual. It can be automated too, but not worth the effort at the moment.